### ISSUE TITLE
expired/time related issues in current master

### ISSUE THREAD

**Issue by mikesnare** [mailboxer/mailboxer#139]
@Roendal FYI, I've found a couple issues in master due to the pull request you accepted from me (#134).
1. Queries and checks against time fields should use Time.zone.now, not Time.now to ensure that the correct zone is being used.  I've got that fixed in my fork.
2. Scopes using Time.zone.now need to be lambdas.  I've got that fixed as well.

I'm going to be doing some more testing over the next day or so before issuing a new pull request.  In the meantime, feel free to revert the pull request you accepted from me or (at least) don't tag a new release with this.  I'll be issuing a pull request soon with the fixes, but only after I've confirmed that everything is working in my project.

Apologies for the mess.
